projects/packages/videopress/src/client/admin/components/edit-video-details/index.tsx

```diff
@@ -132,16 +175,23 @@ const Infos = ( {
 			{ loading ? (
 				<Placeholder height={ 133 } className={ styles.input } />
 			) : (
-				<Input
-					value={ description }
-					className={ styles.input }
-					label={ __( 'Description', 'jetpack-videopress-pkg' ) }
-					name="description"
-					onChange={ onChangeDescription }
-					onEnter={ noop }
-					type="textarea"
-					size="large"
-				/>
+				<>
+					<Input
+						value={ description }
+						className={ styles.input }
+						label={ __( 'Description', 'jetpack-videopress-pkg' ) }
+						name="description"
+						onChange={ onChangeDescription }
+						onEnter={ noop }
+						type="textarea"
+						size="large"
+						rows={ 4 }
```

Comment:
what do you think about increasing the row number to eight to get a better balance between the main section and the sidebar?

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/77539/221237970-3d4ff3ed-5b1d-45a8-9da2-c36c48763101.png">

```suggestion
						rows={ 8 }
```

At least it's something 😅